pandas/core/indexes/datetimes.py

```diff
@@ -1735,7 +1735,33 @@ def freq(self, value):
     is_quarter_start = _field_accessor(
         'is_quarter_start',
         'is_quarter_start',
-        "Logical indicating if first day of quarter (defined by frequency)")
+        """
+        Return a boolean indicating whether the date is the first day of a
```

Comment:
Does 

    Return an indicator for if the date is the first day of the quarter.

fit on a single line?